src/Resources/views/standard_layout.html.twig

```diff
@@ -328,7 +328,7 @@ file that was distributed with this source code.
                         {% block sonata_admin_content %}
 
                             {% block notice %}
-                                {% include ['@SonataCore/FlashMessage/render.html.twig', '@SonataTwig/FlashMessage/render.html.twig'] %}
+                                {% include ['@SonataTwig/FlashMessage/render.html.twig'] %}
```

Comment:
```suggestion
                                {% include '@SonataTwig/FlashMessage/render.html.twig' %}
```

Reply:
Done

Reply:
I think this change can be done on the PR that removes all deprecated code